src/components/Badge/Badge.tsx

```diff
@@ -69,7 +69,11 @@ export const Badge: FC<BadgeProps> = ({
                 {icon && (
                     <span
                         data-test-id="badge-icon"
-                        className={merge([disabled && 'tw-opacity-30', 'tw-flex-none tw-leading-none', 'tw-pr-1'])}
+                        className={merge([
+                            disabled && 'tw-opacity-30',
+                            'tw-flex-none tw-leading-none',
+                            children ? 'tw-pr-1' : '',
```

Comment:
```suggestion
                            'tw-flex-none tw-leading-none',
                            disabled && 'tw-opacity-30',
                            children && 'tw-pr-1',
```

Reply:
Type missmatch when using `children && 'tw-pr-1',`, so we eighter go with `!!children && 'tw-pr-1',` or `children ? 'tw-pr-1' : '',